src/components/targets/CreateTargetForm.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+
+import FormInput from '../common/FormInput';
+import { genderSelectStyle } from '../users/genderSelectStyle';
+import FormSelect from '../common/FormSelect';
+
+const CreateTargetForm = ({ intl }) => {
+  const handleSubmit = () => {};
+
+  const { select_topic } = {};
+
+  const topics = [
+    { value: 'football', label: 'Football' },
+    { value: 'travel', label: 'Travel' },
+    { value: 'politics', label: 'Politics' },
+    { value: 'art', label: 'Art' },
+    { value: 'dating', label: 'Dating' },
+    { value: 'music', label: 'Music' },
+    { value: 'movies', label: 'Movies' },
+    { value: 'series', label: 'Series' },
+    { value: 'food', label: 'Food' },
+  ];
```

Comment:
this should be outside the component given that is static and doesn't need to be rendered each time. Also you can move it to a helpers function if you think it should be re-usable.